src/Event/Event.php

```diff
@@ -188,7 +188,7 @@ public function setResult($value = null)
      * Access the event data/payload.
      *
      * @param string|null $key The data payload element to return, or null to return all data.
-     * @return array|null The data payload if $key is null, or the data value for the given $key. If the $key does not
+     * @return mixed[]|mixed|null The data payload if $key is null, or the data value for the given $key. If the $key does not
```

Comment:
What mixed contains is debatible as to how strict vs verbose. The current annonation would be more explicit. Not neccarily a bad thing.